### ISSUE TITLE
Seems like the predictions are incorrect?

### ISSUE THREAD

**Issue by aabzaliev** [rvcx/stardew-bubbles#1]
hey, first of all - huge thanks for doing this repo and sharing the code! This is amazing. 
I checked 2 random seeds in game, and I think the predictions don't match the game. I guess 1.6 broke it? Is there any chance you can update the app? I am also happy to contrubute if you tell me where to look at, although I never did JS before.

thanks!

**Comment by farkerhaiku** [comment]
can confirm, did approximately 10 seeds so far today and not one has been accurate with the prediction unfortunately. was really looking forward to using this for a min/max run

**Comment by rvcx** [comment]
Yes—the RNG changed in 1.6, which was only released for mobile (which is the main way I play) this month. I haven't quite worked out how the new RNG is applied to splash point generation yet; if you actually have any "with seed X, there were splash points at [(x, y, begin time, end time), ...]" data that would definitely give me a head start.

(I did, however, get the new RNG working for a crummy little Python script I use to pick a "good" seed, with night fairy and traveling cart contents compatible with early community center completion.)
